app/assets/javascripts/channels/synchronized_editor_channel.js

```diff
@@ -23,13 +24,18 @@ $(document).on('turbolinks:load', function () {
 
         received(data) {
           // Called when there's incoming data on the websocket for this channel
-          if (current_user_id !== data['current_user_id']) {
-            CodeOceanEditor.applyChanges(data['delta']['data']);
+          if (data.command === 'editor_change' && current_user_id !== data['current_user_id']) {
```

Comment:
Since you are comparing the `current_user_id !== data['current_user_id']` in all conditions, this could be done first. Then, you could also use a switch statement (optional).